src/com/spikes2212/Cameras/CamerasHandler.java

```diff
@@ -0,0 +1,51 @@
+package com.spikes2212.Cameras;
+
+import java.util.ArrayList;
+
+import com.ni.vision.NIVision;
+import com.ni.vision.NIVision.Image;
+import com.ni.vision.NIVision.ImageType;
+
+import edu.wpi.first.wpilibj.CameraServer;
+
+public class CamerasHandler {
+	private ArrayList<CameraController> cameras;
+	private CameraController currentCamera = null;
+	private Image image;
+	public static int fps = 30;
+
+	public CamerasHandler() {
+		cameras = new ArrayList<CameraController>();
+		this.image = NIVision.imaqCreateImage(ImageType.IMAGE_RGB, 0);
+	}
+
+	public void addCamera(String port) {
+		cameras.add(new CameraController(port, fps));
+	}
+
+	public synchronized void stop() {
+		cameras.forEach(c -> c.stop());
+		currentCamera = null;
+	}
+
+	public synchronized void start(int index) {

```

Comment:
either allow you to start a camera based on its port, or change it so when you add a camera you can specify the index it goes in